opensrp-reveal/src/main/java/org/smartregister/reveal/widget/GeoWidgetFactory.java

```diff
@@ -285,6 +302,12 @@ public void onLocationComponentInitialized() {
         }
     }
 
+    public static boolean isWithinOperationalArea(RevealMapView mapView) {
```

Comment:
is there this method is static, if its static it should be moved to a utils class